ci/jenkins_tests/entry_point.sh

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env bash
+
+# Cause the script to exit if a single command fails.
+set -e
+
+# Show explicitly which commands are currently running.
+set -x
+
+RUN_DOC_TESTS=false
+RUN_TUNE_TESTS=false
+RUN_SGD_TESTS=false
+
+# Parse options
+while [[ $# -gt 0 ]]; do
```

Comment:
I'd do `for key in "$@"; do` instead of `while [[ $# -gt 0 ]]; do ... shift`, since it's better not to `shift` the argument list if you don't need to.